### PR TITLE
Change return_type for get_vm_ip

### PR DESCRIPTION
**Pull request by pkruk**

### CODE REVIEW

File: src/vmshepherd/http/rpc_api.py [hunk 1]
```diff
@@ -113,7 +116,9 @@ def wrap(self, *args, **kwargs):
...
         for vm in preset.vms:
             if vm_id == vm.id:
-                return vm.ip[0]
+                return {'vm_ip': vm.ip[0]}
         # retrieve real time data
         vm_info = await preset.iaas.get_vm(vm_id)
-        return vm_info.ip[0]
+        return {
+            'vm_ip': vm_info.ip[0]
+        }
```
Comment (kwarunek) [inline src/vmshepherd/http/rpc_api.py]: the other way around would be (IMVHO) better change - leave api as is here and match ours internal stuff to this state.

File: src/vmshepherd/errors.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+class VmShepherdError(Exception):
+    message = None
+    details = None
+
+    def __init__(self, message, details=None):
```
Comment (evemorgen) [inline src/vmshepherd/errors.py]:
```suggestion
    def __init__(self, message, details=''):
```

File: src/vmshepherd/errors.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        super().__init__(self.message)
+        self.message = message
+        self.details = details if details else ''
```
Comment (evemorgen) [inline src/vmshepherd/errors.py]: IMO there is no point in this `if` if we apply default value to the `details` kwarg 
```suggestion
        self.details = details
```

File: src/vmshepherd/errors.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        assert self.message is not None
+
+    def __str__(self):
+        if self.details:
+            return 'VmShepherdError: %s, details=%s' % (self.message, self.details)
```
Comment (evemorgen) [inline src/vmshepherd/errors.py]: using `%s` is so Python2 way of doing things, `VmShepherd` runs on Python 3.6. One of it's cool features is [fstrings](https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep498). How about using them? :)

**Pull request merged**